rotkehlchen/chain/evm/contracts.py

```diff
@@ -131,6 +135,115 @@ def decode_event(
         return decode_event_data_abi(tx_log=tx_log, event_abi=event_abi)  # type: ignore
 
 
+def try_download_abi_from_remote(name: str) -> Optional[list[dict[str, Any]]]:
+    """
+    Checks whether abi with the given name exists on remote.
+    If it does, downloads it and adds to globaldb.
+
+    Returns the abi if found and None otherwise.
+    """
+    abi_url = f'{BASE_CONTRACT_DATA_URL}/abi/{name}.json'
+    try:
+        abi_data = request_get(abi_url)
+    except (RemoteError, UnableToDecryptRemoteData) as e:
+        log.error(f'Could not find abi {name} in rotki data repo due to: {str(e)}')
+        return None
+
+    if isinstance(abi_data, list) is False:
+        log.error(f'Abi should be a list. But we got {abi_data}')
+        return None
+
+    with GlobalDBHandler().conn.write_ctx() as write_cursor:
+        write_cursor.execute(
+            'INSERT INTO contract_abi(name, value) VALUES(?, ?)',
+            (name, json.dumps(abi_data, separators=(',', ':'))),
```

Comment:
why are the separators needed here?

Reply:
Afaik this is just the common format abis. Default json separators are `, ` and `: ` and for abis `,` and `:` are used to make it more compact.